src/styles/blocks/card.scss

```diff
@@ -0,0 +1,35 @@
+.card {
+  box-sizing: border-box;
+  display: flex;
+  flex-direction: column;
+  width: 200px;
```

Comment:
According to the [Code Style Rules ❗️](https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules), width should be defined after all the box-model properties. Please move the `width` property below the padding.